src/panels/profile/ha-pick-number-format-row.ts

```diff
@@ -0,0 +1,111 @@
+import "@polymer/paper-item/paper-item";
+import "@polymer/paper-listbox/paper-listbox";
+import {
+  css,
+  CSSResult,
+  customElement,
+  html,
+  LitElement,
+  property,
+  TemplateResult,
+} from "lit-element";
+import "../../components/ha-card";
+import "../../components/ha-paper-dropdown-menu";
+import {
+  CoreFrontendUserData,
+  getOptimisticFrontendUserDataCollection,
+  NumberFormat,
+} from "../../data/frontend";
+import { HomeAssistant } from "../../types";
+import "../../components/ha-settings-row";
+import { formatNumber } from "../../common/string/format_number";
+
+@customElement("ha-pick-number-format-row")
+class NumberFormatRow extends LitElement {
+  @property({ attribute: false }) public hass!: HomeAssistant;
+
+  @property() public narrow!: boolean;
+
+  @property() public coreUserData: CoreFrontendUserData | null | undefined;
+
+  protected render(): TemplateResult {
+    return html`
+      <ha-settings-row .narrow=${this.narrow}>
+        <span slot="heading">
+          ${this.hass.localize("ui.panel.profile.number_format.header")}
+        </span>
+        <span slot="description">
+          ${this.hass.localize("ui.panel.profile.number_format.description")}
+          ${formatNumber(1234567.89, {
+            language: this.hass.language,
+            format: this.coreUserData?.numberFormat,
+          })}
```

Comment:
Can we add this as a secondary line to the `auto` and `system` options instead of in the description?

Reply:
Done!